src/profile/components/Settings/Penalties/Rules.tsx

```diff
@@ -6,127 +6,24 @@ import { useCollapse } from 'common/hooks/collapsible';
-  - Not adhering to the payment deadline. This also results in suspension from all Online's events until payment is completed.
-  
-  The responsible committee can also make a discretionary assessment that benefits the participant.
-  
-  ## Cancellation
-  
-  - In case of illness or other extraordinary events, you will not receive a mark if you unregister 5 hours before the start of the event.
-  - All committees wish that you unregister from events even if you know this will result in a mark. This is so that someone else can become aware of their spot as early as possible.
-  
-  ## Waiting List
-  
-  - If you are on the waiting list, you can unregister right up to the start of the event.
-  - When you're on the waiting list, you understand that you can get a spot in the event at any time and are thus bound by the event rules just like other registered participants.
-  
-  ## Payment
-  
-  - In case of non-payment, you are suspended from all Online's events until the payment is completed.
-  - For paid events, but no show-up, you will not be refunded if deregistration happens after the deadline. If the next person on the waiting list is available, exceptions can be made.
-  
-  ## Behavior
-  
-  - If inappropriate behavior occurs during one of Online's events, you will be financially responsible for any damages and might risk being banned from all of Online's events.
-  
-  ## Corporate Events
-  
-  - For corporate events, the doors open according to the start time of the event. Ten minutes after the doors open, attendees on the waiting list are admitted if there is space. Admission closes 15 minutes after the doors open.
-  - Participants at corporate events are required to respond to the electronic feedback after the events. An email will be sent after the event with a link to the feedback form that must be answered within the given deadline. If a participant does not respond within the deadline, this will result in a mark.
-  - Participants at corporate events must attend all mandatory parts of the event. For company presentations and courses, this means the presentation itself and the course setup. The first 45 minutes followed by mingling are also considered mandatory. If a participant leaves the mandatory part without a valid reason, this will result in a mark.
-  
-  ## Why did I receive a mark?
-  
-  Under "Settings" and "Marks and Suspensions" (https://online.ntnu.no/profile/settings/penalties) you can view your marks and any reasons.
-  
-  If you believe there has been a mistake, please contact the organizer listed for the event. Contact information for the organizing committee is displayed on the event page.
-`;
-
-export const Info = () => {
+export const Info = ({ rules }: IProps) => {
   const rulesId = 'rules';
 
   const { asPath } = useRouter();
   const hash = asPath.split('#')[1];
 
   const [collapsed, toggleCollapse] = useCollapse(hash !== rulesId);
 
+  const ruleset = rules.length
+    ? rules[0].content
```

Comment:
Kan du passe på at den henter ut den som faktisk er gyldig nå?

Reply:
From the data I have tested with, it will always take the latest one. But I can sort by version number instead?

Reply:
Rekkefølgen til endepunktet er `valid_from`, som kan være i fremtiden, så ta første som ikke er i fremtiden